README.md

```diff
@@ -126,3 +128,16 @@ module "terraform-dome9-awp-azure" {
 <!-- END_TF_DOCS -->
 
 ## FAQ & Troubleshooting
+### Centralized Offboarding with sse_cmk_encrypted_disks_scan Enabled
+
+When centralized offboarding is performed and sse_cmk_encrypted_disks_scan is enabled, AWP Keys must be deleted manually. This should be done prior to initiating the offboarding process.
```

Comment:
"AWP Keys must be deleted manually" - not really "must".
"This should be done prior to initiating the offboarding process." - can be done also after.

Just write something like "If you want to avoid the retention time of the keys, you can manually..."